src/vivarium_gates_nutrition_optimization/components/children.py

```diff
@@ -38,7 +38,11 @@ def setup(self, builder: Builder) -> None:
     def on_collect_metrics(self, event: Event):
         pop = self.population_view.get(event.index)
         new_birth_mask = (
-            (pop["pregnancy_term_outcome"] == models.FULL_TERM_OUTCOME)
```

Comment:
I thought we said we didn't want to store the live-birth/stillbirth distinction in this column because maternal hemoglobin at the time of birth can impact whether or not there is a live birth. Is this not the case?